src/test/java/org/folio/rest/impl/CheckinReceivingApiTest.java

```diff
@@ -337,8 +337,14 @@ void testReceiveOngoingOrder() {
     assertThat(pieceUpdates, hasSize(receivingRq.getTotalRecords()));
     assertThat(polSearches, hasSize(pieceIdsByPol.size()));
 
-    // check no status updates were performed
+    // check no status updates were performed and POL remained ongoing
     assertThat(polUpdates, nullValue());
+    polSearches.forEach(pol -> {
+      PoLine poLine = pol.mapTo(PoLineCollection.class).getPoLines().get(0);
+      assertThat(poLine.getCheckinItems(), is(false));
+      assertThat(poLine.getReceiptStatus(), is(PoLine.ReceiptStatus.ONGOING));
+      assertThat(poLine.getReceiptDate(), is(notNullValue()));
+    });
```

Comment:
please check if all branchers of source code is covered with unit tests

Reply:
![image](https://github.com/folio-org/mod-orders/assets/144257054/36d0cdcf-730c-413a-bdff-11a7f71a8e77)